extensions/amp-analytics/0.1/requests.js

```diff
@@ -436,40 +401,22 @@ function getExtraUrlParams(
 }
 
 /**
- * Handle the params map and form the final extraUrlParams string
- * @param {!./variables.VariableService} variableService
- * @param {!Object} params
- * @return {string}
- */
-function getExtraUrlParamsString(variableService, params) {
-  const s = [];
-  for (const k in params) {
-    const v = params[k];
-    if (v == null) {
-      continue;
-    } else {
-      const sv = variableService.encodeVars(k, v);
```

Comment:
encodeVars was needed to keep the macro unchanged. it's now unneeded because all macros are resolved.